.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -29,21 +22,28 @@ Acceptance Criteria is a set of test scenarios specific to this particular issue
 - [ ] ...
 
 **Risks:**
-- [ ] ...
-- [ ] ...
-- [ ] ...
-- [ ] ...
+- ...
+- ...
+- ...
+- ...
```

Comment:
While we're here... would it make sense to make it only one list item? 4 seems like more to delete since we usually only have 1 or 2...

Reply:
Done.